### ISSUE TITLE
[ v4 ] <Miss> Component Name.

### ISSUE THREAD

**Issue by sunyang713** [ReactTraining/react-router#3885]
Hi, I really don't like the name "Miss" for the component that handles a "No Match." Why can't you call it <NoMatch>, like you have typically used for v2/3? I feel like "Miss" is too vague and non-descriptive of a term. Isn't it very specifically the 'route' that handles a "No Match?" In your v4 examples, you pass in a component named "NoMatch." If it's an issue with the conflicting names between the component and the React-Router provided 'Route,' I think you should make the react-router name the more proper, precise one. The passed-in component could be more appropriately called "NotFound," as it refers to the visual component that presents "Whoops, not found!" to the user.

**Comment by ryanflorence** [comment]
import it as any name you want :) we love match/miss though

**Comment by timdorr** [comment]
Good point. You're free to do `import { Match as Yup, Miss as Nope } from 'react-router'`.
